barretenberg/cpp/src/barretenberg/vm/tests/AvmMini_execution.test.cpp

```diff
@@ -65,17 +66,17 @@ TEST_F(AvmMiniExecutionTests, basicAddReturn)
     auto instructions = Deserialization::parse(bytecode);
 
     // 2 instructions
-    EXPECT_THAT(instructions, testing::SizeIs(2));
+    EXPECT_THAT(instructions, SizeIs(2));
 
     // ADD
     EXPECT_EQ(instructions.at(0).op_code, OpCode::ADD);
 
     auto operands = instructions.at(0).operands;
```

Comment:
It doesn't matter too much (since it's a test) but here you are doing a copy. Ideally you don't even need this (given the other suggestion) but if you prefer to keep it then better to have `const T&` (or in the codebase's style `T const&`?)

Reply:
Thanks. This is good habit to not use more memory than needed. I will probably implement the above suggestion.